app/views/welcome/index.html.erb

```diff
@@ -1,16 +1,12 @@
 <% if current_user %>
     <% if current_user.instructor %>
-        User is an instructor
-
-
-
-
-
-
-
-
-
-
+        <div class="container">
+            <h2>Available cases</h2>
+            <% @chat_rooms.each do |room| %>
+                <small>Created by: <%= @user.find(room.user_id).name %> at <%= room.created_at %></small>
```

Comment:
There's no need to find the user directly. Calling `room.user.name` will get you what you want.